src/firmware_new/boards/frankie_v1/main.c

```diff
@@ -621,6 +702,37 @@ void StartDefaultTask(void *argument)
     /* USER CODE END 5 */
 }
 
+/* USER CODE BEGIN Header_test_msg_update */
+/**
+ * @brief Function implementing the testMsgUpdate thread.
```

Comment:
Add whitespace between your @s to keep it consistent with the rest of the comments.

Reply:
this is also autogen 😢 cube will just wipe any changes